chemprop/args.py

```diff
@@ -601,6 +601,13 @@ def process_args(self) -> None:
         if not self.bond_feature_scaling and self.bond_features_path is None:
             raise ValueError('Bond descriptor scaling is only possible if additional bond features are provided.')
 
+        # normalize target weights to sum to one
+        if self.target_weights is not None:
+            avg_weight = sum(self.target_weights)/len(self.target_weights)
```

Comment:
I am not sure (see main comment) why this would not just be the sum

Reply:
Thinking about it longer, it might be better to normalize via the averages (to not make the gradients smaller as they need to be), so that the average of the weights is 1, not the sum. This is fine, but then line 604 (the comment) should be changed

Reply:
Good catch with the comment!

Using the average is also better than sums because it makes it deviate less from and more consistent with the past and default behavior where weights would be [1,1,1] instead of [.33,.33,.33].

Reply:
Yes, totally agree, I was just confused by the comment